systemFlake.nix

```diff
@@ -1,12 +1,13 @@
 { flake-utils-plus }:
 
 { self
-, defaultSystem ? "x86_64-linux"
+, defaultSystem ? "x86_64-linux" # will be deprecated soon use defaultHostAttrs.system instead
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
 , nixosConfigurations ? { }
 , sharedExtraArgs ? { }
+, defaultHostAttrs ? { }
```

Comment:
Not a fan of the name, how do these two sound to you?
`hostDefaults`
`defaultHostConfig`

Reply:
I like both, I vote for `hostDefaults`

Reply:
But both names don't communicate how the modules under `defaultHostAttrs` is shared. default kind of sounds like it gets overridden. The examples are good though, so I don't think it will be much of an issue.

Reply:
```
nix-repl> _ = x: op: y: op x y

nix-repl> __or = lhs: rhs: if lhs != null then lhs else rhs

nix-repl> or 1 2
error: syntax error, unexpected OR_KW
```

I conclude `or` is not an operator and for keywords it doesn't work.

Reply:
Done: renamed to `hostDefaults` & `hosts` accordingly